README.md

```diff
@@ -110,8 +110,9 @@ Individual nodes within the `nodes` property can have the following structure:
 
 | Property    | Type   | Description                     |
 | ----------- | ------ | ------------------------------- |
-| `label`     | string | **Required**. The node's label. |
+| `label`     | mixed  | **Required**. The node's label. |
 | `value`     | mixed  | **Required**. The node's value. |
 | `children`  | array  | An array of child nodes.        |
 | `className` | string | A className to add to the node. |
 | `icon`      | mixed  | A custom icon for the node.     |
+| `disabled`  | bool   | Disable node.                   |
```

Comment:
I would prefer `disabled` to appear above `icon` and below `className` to be alphabetically sorted below the required properties.